homeassistant/components/notify/hangouts.py

```diff
@@ -0,0 +1,76 @@
+"""
+Hangouts notification service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.hangouts/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.notify import (ATTR_TARGET, PLATFORM_SCHEMA,
+                                             NOTIFY_SERVICE_SCHEMA,
+                                             BaseNotificationService,
+                                             ATTR_MESSAGE)
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_DEFAULT_CONVERSATIONS = 'default_conversations'
+CONF_CONVERSATION_ID = 'id'
+CONF_CONVERSATION_NAME = 'name'
+
+DOMAIN = 'hangouts'
+DEPENDENCIES = [DOMAIN]
+
+TARGETS_SCHEMA = vol.All(
+    vol.Schema({
+        vol.Exclusive(CONF_CONVERSATION_ID, 'id'): cv.string,
+        vol.Exclusive(CONF_CONVERSATION_NAME, 'id'): cv.string
```

Comment:
Is 'id' the correct 'group of exclusion'?

Reply:
yes and no. the group is correct, but the naming is bad